images/entrypoint.sh

```diff
@@ -178,5 +178,7 @@ fi
 
 echo "Entering sleep... (success)"
 
-# Sleep forever.
-sleep infinity
+# Sleep until container is stopped.
```

Comment:
in a follow up it would be great to make this "sleep until the heat death of the universe" approach optional, so the container can be used as an init container more easily.

Reply:
Please look at the new Version at https://github.com/intel/multus-cni/pull/259 - I have added this as an option.